_gsocblogs/2022/blog_CompilerResearch-ManishKausik.md

```diff
@@ -0,0 +1,28 @@
+---
+project: Compiler-Research
+title: Add Initial Integration of Clad with Enzyme 
+author: Manish Kausik H
+photo: blog_authors/ManishKausik.jpg
+date: 24.07.2022
+year: 2022
+layout: blog_post
+logo: compiler-research-logo.png   
+intro: |
+  A blog that summarises my experience with GSoC so far and also gives a brief of the work I've done. 
+---
+## What Interested me to apply for this Project?
+Ever since as I was introduced to Compiler Design as a part of my university course work, I have been fascinated by this subject and was always eager to explore more about it. Naturally
```

Comment:
```suggestion
Ever since as I was introduced to compiler design as a part of my university course work, I have been fascinated by this subject and was always eager to explore more about it. Naturally
```